src/test/java/edu/ucsb/cs156/frontiers/annotations/WithInstructorCoursePermissions.java

```diff
@@ -0,0 +1,17 @@
+package edu.ucsb.cs156.frontiers.annotations;
+
+import org.springframework.security.test.context.support.WithMockUser;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * This annotation interacts with the mock CourseSecurity annotation. It adds authorities that tell the CourseSecurity annotation
+ * used in testing that the user has permission to manage the course being accessed. It should be used on methods that have the
+ * <i>@PreAuthorize("@CourseSecurity.hasManagePermissions")</i> annotation.
+ * It cannot be combined with <i>@WithMockUser</i>.
```

Comment:
I wonder if it might be good to say:

This should not be combined with @WithMockuUser, because this annotation implies @WithMockUser("ROLE_INSTRUCTOR")